content/issues/2023-05-28-mariadb.md

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Known Issue: MariaDB Check and Repair'
+date: '2023-05-28 12:00:00Z'
```

Comment:
Update the publish date - if it's more than 24 hours ago the RSS bot won't pick it up automatically.